src/components/SearchBox.tsx

```diff
@@ -1,75 +1,51 @@
 import React from 'react'
-import { StatelessComponent, ReactElement, useEffect, useState } from 'react';
-import { navigateTo } from "gatsby-link"
+import { StatelessComponent, ReactElement } from 'react';
+import { navigate } from "gatsby"
```

Comment:
`navigateTo` and `navigate` do the same thing, but `navigateTo` is deprecated so may as well stop using that while we're at it.